src/utils/tokens.js

```diff
@@ -0,0 +1,31 @@
+
+import sendJson from '../tmp_fetch_send_json'
+import { ACCOUNT_HELPER_URL } from './wallet'
+import * as Sentry from '@sentry/browser'
+import { wallet } from './wallet'
+
+export const getLikelyContracts = async (accountId) => (
+    await logAndIgnoreError(sendJson('GET', `${ACCOUNT_HELPER_URL}/account/${accountId}/likelyTokens`))
+)
+
+export const getMetadata = async (contract, accountId) => {
+    const account = await wallet.getAccountBasic(accountId)
+
+    let metadata = await logAndIgnoreError(account.viewFunction(contract, 'ft_metadata'))
+    return {
+        contract,
+        metadata
+    }
+}
+
+export const getBalanceOf = async (contract, accountId) => {
+    const account = await wallet.getAccountBasic(accountId)
+
+    let balance = await logAndIgnoreError(account.viewFunction(contract, 'ft_balance_of', { account_id: accountId }))
+    return {
+        contract,
+        balance
+    }
+}
+
+const logAndIgnoreError = (promise) => promise.catch(error => (console.warn(error), Sentry.captureException(error)));
```

Comment:
I'm curious about this indirection and lazy attachment of a .catch() handler -- why not `.catch(logError)` on each async call that we want to add a catch handler to? `logAndIgnoreError(...)` is more characters to type than `.catch(logError)`, it requires wrapping a function call rather than chaining on it, and the code doesn't read sequentially because of it being handled via a wrapper function rather than being chained on the call. 
 I don't think it's really any DRYer (we repeat `logAndIgnoreError` the same # of times we would repeat `.catch(logError)`), and it adds an extra layer of indirection that doesn't seem to add any value over .catch(logError)` unless I am missing something here.

Reply:
Done